src/mdbExtensionController.ts

```diff
@@ -422,10 +438,31 @@ export default class MDBExtensionController implements vscode.Disposable {
       name: 'MongoDB Shell',
       env: { MDB_CONNECTION_STRING: mdbConnectionString }
     });
+
+    if (this.isSslConnection(activeConnectionModel)) {
+      mdbTlsOptionsString = '--tls';
```

Comment:
To test dev mode with self-signed certificates use `mdbTlsOptionsString = '--tls --tlsAllowInvalidCertificates';`

So the question stays here, how to test real case, the one without `--tlsAllowInvalidCertificates` flag.